client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -244,8 +244,15 @@ export const handlePlatformCheckoutEmailInput = ( field, api ) => {
 		return pattern.test( value );
 	};
 
-	// Check the initial value of the email input and trigger input validation.
-	if ( validateEmail( platformCheckoutEmailInput.value ) ) {
+	// Prevent show platform checkout iframe if the page comes from
+	// the back button on platform checkout itself.
+	const searchParams = new URLSearchParams( window.location.search );
+
+	if (
+		'true' !== searchParams.get( 'skip_platform_checkout' ) &&
```

Comment:
Not really necessary but I think it'd be nice to remove the param from the URL (using `history.replaceState()`) if the `skip_platform_checkout` param is found.

Reply:
Good suggestion, added on c74f0343.